plugin/evm/customtypes/aliases_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
```

Comment:
These were originally `.` imported in the individual test files, but there is now a conflicting function name so they can't be.